### DISCUSSION TITLE
feat: Select multiple resources to act on

### DISCUSSION THREAD

**Discussion by Keyruu** [Ramilito/kubectl.nvim#433] [General]
I would love to select multiple pods and then delete all of them. Is that possible?

**Comment by Ramilito** [comment]
Not possible right now but it's a great idea and something I have wanted to have. I'll reword the title as a feature request!

**Reply by Keyruu** [reply]
Awesome thank you!

**Comment by Ramilito** [comment]
You can now select items using tab! #465
